binder/getting-started/code-snippets/mit_license_text.json

```diff
@@ -0,0 +1,13 @@
+{
+  "display_name": "MIT license text",
+  "metadata": {
+    "language": "Markdown",
+    "code": [
+      "### Copyright",
+      "",
+      "Copyright \u00a9 2020 <COPYRIGHT-HOLDER>. This notebook and its source code are released under the terms of the [MIT License](https://opensource.org/licenses/MIT)."
+    ],
+    "description": "See https://opensource.org/licenses/MIT"
+  },
+  "schema_name": "code-snippet"
+}
```

Comment:
I don't know if this is a good example, it will show as markdown, but if you want to add to a script or something you will get a warning, maybe having something about pure code (can be used in both python and notebook) and code that help on notebooks (more like visualization)?

Reply:
Right, I chose markdown on purpose to raise awareness why the snippet language type matters:

> Try to insert the markdown snippet [into a code cell]. A warning should be displayed indicating that the snippet is not compatible with the cell's type. 

I do agree a markdown snippet itself is of limited value. However, we already have a basic Python code example included in the tour and I therefore wasn't aiming at providing a comprehensive set of snippets. I would consider an establishing a collection of production-is snippets that can be easily published and imported a separate effort. This would be similar to Node-REDs approach https://flows.nodered.org, where anybody can publish and import.